source/activity.txt

```diff
@@ -0,0 +1,138 @@
+:ref:`View Application Logs <view-logs>`.
+
+A log entry describes a single application event of a given type. Each
+entry contains information about the event and how the system responded.
+For example, a log may include the IP address that issued a request as
+well as a summary of the data access permissions that were evaluated and
+assigned to serve the request.
+
+Apps log the following event types:
+
+- :ref:`Authentication <logs-authentication>`, including user
+  creation, login, and deletion.
+
+- :ref:`Change Stream <logs-change-stream>`, including any time a user
+  opens or closes a stream of change events.
+
+- :ref:`Device Sync <logs-sync>`, including all events related to
+  data synchronization between client devices and MongoDB Atlas.
+
+- :ref:`Endpoint <logs-endpoint>`, including any time a request is made
+  to a Data API endpoint.
+
+- :ref:`Function <logs-function>`, including both Atlas Functions
+  as well as individual queries to linked MongoDB Atlas data sources made using the
+  MongoDB service of the a Realm SDK.
+
+- :ref:`Service <logs-service>`, including HTTPS endpoints and
+  service action calls issued from the a Realm SDK.
+
+- :ref:`Schema <logs-schema>`, including any events related to
+  changes to an application's schema.
+
+- :ref:`Trigger <logs-trigger>`, including Database Triggers,
+  Authentication Triggers, and Scheduled Triggers.
+
+Error Logs
+~~~~~~~~~~
+
+All App Services log entries have one of two possible statuses:
+
+- ``OK``, which represents a normal event that succeeded without an error.
+
+- ``Error``, which represents an event that did not run successfully for
+  any reason.
+
+For example, App Services would log an error for any of the following
+events:
+
+- You attempt to access data stored in Atlas for which there is no
+  applicable rule.
+
+- You throw or fail to handle an error or promise rejection in an Atlas
+  Function.
+
+- You call ``context.services.get()`` for a service which does not exist.
+
+Log Filters
+~~~~~~~~~~~
+
+For performance reasons, App Services limits individual queries to a
+maximum of 100 log entries per page. You can filter entries by type,
+status, timestamp, user, and request ID to return only logs that are
+relevant to your query.
+
+Log Lines
+~~~~~~~~~
+
+Functions can log information using JavaScript's ``console.log()``
+method. App Services stringifies each console log and stores each string
+as a single line. App Services truncates lines to 512 bytes in length.
+For ASCII character sets, this translates to 512 characters; depending
+on the character set you use, you may see truncation at lower character
+counts.
+
+App Services saves only the first 25 log lines for a given log entry.
+
+Log Retention
+~~~~~~~~~~~~~
+
+App Services retains logs for {+log-retention-time+}, after which they
+are deleted.
+
+If you require logs older than {+log-retention-time+}, you can
+automatically :ref:`forward logs <forward-logs>` to another service. You
```

Comment:
Not seeing this page in the left toc after clicking link

Reply:
Accidentally left it off! Fixed.